### ISSUE TITLE
ecr-viewer view more button is missing aria-expanded attribute

### ISSUE THREAD

**Issue by BobanL** [CDCgov/phdi#1754]
## Action Requested
- `view more`/ `view less` buttons should contain [aria-expanded](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) attribute depending on if it is open or closed

## Acceptance Criteria
- `view more` and `view less` contain aria-expanded attribute.

## Additional Information
- https://cdc-devtools.dequecloud.com/issues/669a8cd8-603e-4084-adcf-5f7fbbe92098
- https://docs.deque.com/issue-help/1.0.0/en/aria-state-property-missing
